app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -239,23 +223,61 @@ textarea {
   overflow-x: hidden; /* fix for FF adding an extra row to textareas */
   &.no-stretch {
     resize: vertical;
-    min-height: 34px; // prevent firefox from resizing smaller then one line height
+    min-height: 26px; // prevent firefox from resizing smaller then one line height
   }
 }
 
+.glyphicon-info-sign{
```

Comment:
not sure, can you verify (search for all places with info sign..) maybe the classes selections etc?